com.unity.cinemachine/Runtime/Core/CinemachineVirtualCameraBase.cs

```diff
@@ -801,7 +801,12 @@ class OnDomainReload
         { 
             static OnDomainReload() 
             {
+#if UNITY_2023_1_OR_NEWER
+                var vcams = FindObjectsByType<CinemachineVirtualCameraBase>
+                    (FindObjectsInactive.Include, FindObjectsSortMode.InstanceID);
```

Comment:
Sorry, I was wrong to say earlier that the order needs to be determinate here.  It doesn't.  Better to use the unsorted version.